Vialer/CellWithTextField.m

```diff
@@ -0,0 +1,24 @@
+//
+//  CellWithTextField.m
+//  Vialer
+//
+//  Created by Harold on 22/06/15.
+//  Copyright (c) 2015 VoIPGRID. All rights reserved.
+//
+
+#import "CellWithTextField.h"
+
+@implementation CellWithTextField
```

Comment:
Consider changing classname to  EditNumberTextFieldTableViewCell or something similar. Describes in a little bit more detail what it is and what it does.